### ISSUE TITLE
Wrong unique user count

### ISSUE THREAD

**Issue by sinnbeck** [andreaselia/laravel-analytics#24]
The unique user count seems to be wrong. It gets the count of how many page visits the last visitor had.

Maybe do something like this to keep it simple
```php
    'value' => PageView::query()
                    ->scopes(['filter' => [$this->period]])
                    ->groupBy('session')
                    ->pluck('session')
                    ->count(),
```
Let me know if you want a PR.

edit: seems this PR broke it: https://github.com/andreaselia/laravel-analytics/pull/22

**Comment by tomirons** [comment]
If you can push up a PR that would be fantastic!

**Comment by tomirons** [comment]
This was fixed with #25.
